test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -0,0 +1,42 @@
+import { Selector } from 'testcafe';
+import BasePageObject from './BasePageObject';
+import BaseFormObject from './components/BaseFormObject';
+
+export default class IdentityPageObject extends BasePageObject {
+  constructor (t) {
+    super(t);
+    this.form = new BaseFormObject(t);
+  }
+
+  fillIdentifierField(value) {
+    return this.form.setTextBoxValue('identifier', value);
+  }
+
+  getIdentifierValue() {
+    return this.form.getTextBoxValue('identifier');
+  }
+
+  clickNextButton() {
+    return this.form.clickSaveButton();
+  }
+
+  waitForErrorBox() {
+    return this.form.waitForErrorBox();
+  }
+
+  waitForIdentifierError(){
```

Comment:
why need both `waitForIdentifierError` and `waitForErrorBox`

Reply:
@haishengwu-okta You can have identifier error without box error (empty field on focus out)